pyseir/inference/model_fitter.py

```diff
@@ -170,7 +170,7 @@ def set_inference_parameters(self):
 
         if len(self.fips) == 5:
             # Setting
-            idx_enough_cases = np.argwhere(np.cumsum(self.observed_new_cases) >= 3)[0][0]
+            idx_enough_cases = np.argwhere(np.cumsum(self.observed_new_cases) >= 2)[0][0]
```

Comment:
Would love to pull this `2` out as a constant, just to give an opportunity to comment if possible